shell/components/CruResource.vue

```diff
@@ -427,6 +427,83 @@ export default {
 
     shouldProvideSlot(slot) {
       return slot !== 'default' && typeof this.$slots[slot] === 'function';
+    },
+
+    formatError(err) {
+      if ( typeof err === 'string') {
+        return err;
+      }
+      let str = '';
+      const msg = !!err?.message ? ` ${ err?.message }` : '';
+
+      if ( err?.code === 'ActionNotAvailable' ) {
+        str = this.t('errors.actionNotAvailable');
+      } else if ( err?.status === 422 ) {
+        str = this.t('errors.failedInApi');
+        let something = false;
+
+        if ( err?.fieldName ) {
+          str += ` '${ err?.fieldName }'${ msg }`;
```

Comment:
Is this how it was already done in Ember? If so, I'll accept this.

In general it tends to be a bad idea to modify/concatenate translated strings since not all languages are read left to right top to bottom.

Reply:
Got it, given it's a direct port I don't expect you to make any changes.

In general we should make better use of parameters in the translation strings.